csv2geojson.py

```diff
@@ -0,0 +1,37 @@
+import sys
+import csv
+import json
+
+argv = sys.argv[1:]
+if len(argv) == 0:
+    print("no csv file specified")
+    sys.exit(2)
+csvfile = argv[0]
```

Comment:
Naming: this is not a file, but a path.

Better approach of this: using Pathlib. Then it's `csvfile = Path(argv[0])`, and `with csvfile.open(...) as ...`.